test/self_driving_e2e/model_server_test.cpp

```diff
@@ -12,17 +12,16 @@ namespace noisepage::modelserver {
 
 /**
  * @warning Running this test requires external dependency to be located at specific paths.
- *  The environment BUILD_ABS_PATH needs to be set to be the build directory such that:
- *      1. BUILD_ABS_PATH/../script/model/model_server.py is executable
```

Comment:
This should be back. And actually, we probably should document why it's pretty tricky to use relative path and we have to use the environment variable here. So that someone else in the future does not need to repeat this effort.